python/cudf/cudf/core/frame.py

```diff
@@ -1891,8 +1891,17 @@ def __rmatmul__(self, other):
         return self.dot(other, reflect=True)
 
     # Unary logical operators
+    @_cudf_nvtx_annotate
```

Comment:
Can we annotate everything in this file, or drop the annotation here? Just want consistency. Fine to do that in a separate PR.

Reply:
Annotated in this PR 👍